src/sagas/wallet.js

```diff
@@ -215,20 +221,17 @@ export function* startWallet(action) {
       // the feature flag
       yield call(featureFlags.ignoreWalletServiceFlag.bind(featureFlags));
 
-      // Restart the whole bundle to make sure we clear all events
-      walletHelpers.reloadElectron();
+      // Cleanup all listeners
+      yield cancel(threads);
+
+      // Yield the same action so it will now load on the old facade
+      yield put(action);
```

Comment:
Should we `return` here, so it can start this action again with the old facade?

Reply:
We don't, since this is a generator, it will end after the last yield